index.js

```diff
@@ -32,7 +32,7 @@ if (argv.gasprice < 1 || argv.gasprice > 1000) throw "--gasprice must be between
   // Example EMP Parameters. Customize these.
   const empParams = {
     expirationTimestamp: "1640995200", // 01/01/2022 @ 0:00 (UTC)
-    collateralAddress: "0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2", // WETH
+    collateralAddress: "0xd0a1e359811322d97991e03f863a0c30c2cf029c", // Kovan WETH address
```

Comment:
Maybe add a comment to note the mainnet WETH address so it's easy to swap in?

Reply:
Good idea. Added in a comment